template/base/scripts/program/dump.mjs

```diff
@@ -10,24 +11,30 @@ const rpc = process.env.RPC ?? 'https://api.mainnet-beta.solana.com';
 const outputDir = getExternalProgramOutputDir();
 await dump();
 
-/** Dump external programs binaries if needed. */
+/** Dump external programs binaries and accounts if needed. */
 async function dump() {
-  // Ensure we have some external programs to dump.
-  const addresses = getExternalProgramAddresses();
-  if (addresses.length === 0) return;
+  // Ensure we have some external accounts to dump.
+  const programs = getExternalProgramAddresses();
+  const accounts = getExternalAccountAddresses();
+  const external = [
+    ...programs.map((program) => [program, 'so']),
+    ...accounts.map((account) => [account, 'json']),
+  ];
+
+  if (external.length === 0) return;
   echo(`Dumping external accounts to '${outputDir}':`);
 
   // Create the output directory if needed.
   $`mkdir -p ${outputDir}`.quiet();
 
   // Copy the binaries from the chain or warn if they are different.
   await Promise.all(
-    addresses.map(async (address) => {
-      const binary = `${address}.so`;
+    external.map(async (address, extension) => {
```

Comment:
I think this won't work because it'll think that extension is the index of the array. You need to use the following instead:

```suggestion
    external.map(async ([address, extension]) => {
```